WcaOnRails/config/locales/en.yml

```diff
@@ -51,16 +51,16 @@ en:
     minx: "Megaminx"
     pyram: "Pyraminx"
     sq1: "Square-1"
-    clock: "Rubik's Clock"
+    clock: "Clock"
```

Comment:
The order of the events is wrong here. The side events are sorted alphabetically, so Clock should be before Megaminx.

Reply:
For the record, the order of stuff in the yml file doesn't actually affect anything (ordering is driven by a `rank` int in the database, which I believe @coder13 did correctly update), but I do agree that it would be nice to reorder them here just for parallelism.